platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/ConfigMappings.java

```diff
@@ -49,7 +49,16 @@ private ConfigMappings() {}
             new ConfigMapping("event.eventStreamQueueCapacity", "eventStreamQueueCapacity"),
             new ConfigMapping("event.eventsLogPeriod", "eventsLogPeriod"),
             new ConfigMapping("event.eventsLogDir", "eventsLogDir"),
-            new ConfigMapping("event.enableEventStreaming", "enableEventStreaming"));
+            new ConfigMapping("event.enableEventStreaming", "enableEventStreaming"),
+            new ConfigMapping("metrics.halfLife", "halfLife"),
+            new ConfigMapping("metrics.csvWriteFrequency", "csvWriteFrequency"),
+            new ConfigMapping("metrics.csvOutputFolder", "csvOutputFolder"),
+            new ConfigMapping("metrics.csvFileName", "csvFileName"),
+            new ConfigMapping("metrics.csvAppend", "csvAppend"),
+            new ConfigMapping("metrics.disableMetricsOutput", "disableMetricsOutput"),
+            new ConfigMapping("prometheus.endpointEnabled", "endpointEnabled"),
+            new ConfigMapping("prometheus.endpointPortNumber", "endpointPortNumber"),
```

Comment:
Is that correct? in the config you replaced `prometheusEndpointEnabled` and not `endpointEnabled`?

Reply:
Idea Magic - thanks for the hint